workitem_blackbox_test.go

```diff
@@ -106,22 +106,21 @@ func TestReorderWorkItem(t *testing.T) {
 	_, result1 := test.CreateWorkitemCreated(t, svc.Context, svc, controller, &payload1)
 	_, result2 := test.CreateWorkitemCreated(t, svc.Context, svc, controller, &payload1)
 	_, result3 := test.CreateWorkitemCreated(t, svc.Context, svc, controller, &payload1)
+	defer test.DeleteWorkitemOK(t, nil, nil, controller, *result1.Data.ID)
```

Comment:
Hi @DhritiShikhar ! Good job! Except that I would put the defer call right after each resource's acquisition.

```go
_, result1 := test.CreateWorkitemCreated(t, svc.Context, svc, controller, &payload1)
defer test.DeleteWorkitemOK(t, nil, nil, controller, *result1.Data.ID)
_, result2 := test.CreateWorkitemCreated(t, svc.Context, svc, controller, &payload2)
defer test.DeleteWorkitemOK(t, nil, nil, controller, *result2.Data.ID)
_, result1 := test.CreateWorkitemCreated(t, svc.Context, svc, controller, &payload3)
defer test.DeleteWorkitemOK(t, nil, nil, controller, *result3.Data.ID)
```

I know, code-wise this doesn't *look* as nice as your layout but it is the safer method.